examples/6_add_image_transforms.py

```diff
@@ -0,0 +1,46 @@
+from pathlib import Path
```

Comment:
I feel like I lost sight of a comment from Remi about adding a dosctring here like the other examples? Anyway, I agree on that. Can we do it?